lib/connectors/gitlab/connector.rb

```diff
@@ -33,15 +33,19 @@ def self.configurable_fields
         }
       end
 
-      def initialize(local_configuration = {})
-        super(local_configuration)
+      def initialize(local_configuration:, connector_settings:)
```

Comment:
The named parameter with no default value implies that we always need to specify it. What if we don't have any settings to store in the yaml? Or do we assume that there's at least some credentials, always?

Reply:
But it's making the code longer, not just by the `{}`, but by the whole length of `local_configuration: {}`. =)

Reply:
I think that if we decided to pass both in as empty hashes, it would be confusing if the params were not named. However, with named params and no defaults, it's just not providing an option of just calling a method with no params at all.

Reply:
Vote for 

```
def initialize(local_configuration: {}, connector_settings: {})
```